pkg/infra/tracing/tracing.go

```diff
@@ -194,10 +194,8 @@ func (s OpentracingSpan) End() {
 	s.span.Finish()
 }
 
-func (s OpentracingSpan) SetAttributes(kv ...attribute.KeyValue) {
-	for k, v := range kv {
-		s.span.SetTag(fmt.Sprint(k), v)
```

Comment:
I'm sorry, but might misunderstand something. Couldn't you just have read each key and value from the `v` variable? Or do you have to type cast to be able to get something that makes sense out of it?

Reply:
with key it would work but with value you need to know the type to get the value, which I thought would be easier this way

Reply:
I can pass the type and attribute.KeyValue, that is another option

Reply:
@marefr due to the fact that this code will be removed soon I would leave it like this, wdyt?

Reply:
Sure it's fine. It was just that otel had some nicer/more typesafe API's, but we can switch to that in the future totally fine.